### DISCUSSION TITLE
空間資訊框線密封問題

### DISCUSSION THREAD

**Discussion by HutakiHare** [depositar-io/ckanext-data-depositario#23] [General]
在空間資訊中使用 poly line 時產生的框可以是有漏洞的，但範圍的的圈取並不應有缺口，反之，也可以畫出超過框的線，或許可以規定開頭的點必須與結束的點重合，使框範圍變得更間單與合理 ? (以前使用 labelme 工具時有看到類似功能)

**Comment by u10313335** [comment]
感謝建議。目前 depositar 資料集的「空間範圍」是指英文的 spatial coverage 之意，不一定要封閉。
